src/common/FPSTimer.cpp

```diff
@@ -57,3 +57,25 @@ void FPSTimer::update(double elapsedTime, double renderingTime, int logCount)
     mRecordFpsFrequencyCursor++;
 }
 
+bool FPSTimer::variance(float *avg)
+{
+    for (int i = 0; i < NUM_HISTORY_DATA; i++)

```

Comment:
- It's better to hold the assignment of avg until the result is confimed to be valid.
- Increment is not a good way. If do so, avg should be reset to 0 firstly inside this funtion.

Reply:
Done